lib/jsonapi/mime_types.rb

```
@@ -5,5 +5,7 @@ module JSONAPI
 Mime::Type.register JSONAPI::MEDIA_TYPE, :api_json
 
 ActionDispatch::ParamsParser::DEFAULT_PARSERS[Mime::Type.lookup(JSONAPI::MEDIA_TYPE)] = lambda do |body|
-  JSON.parse(body)
+  data = JSON.parse(body)
+  data = {:_json => data} unless data.is_a?(Hash)
```

This isn't ever reached in the current test suite, and I think `JSON.parse` will raise an error if it can't convert to a hash. It seems like we would also want to raise an error if we ever get back a result that isn't a Hash. Is there something I'm missing here?